src/main/java/appeng/items/parts/PartType.java

```diff
@@ -132,6 +133,9 @@ public boolean isCable() {
 
     QuartzFiber(140, EnumSet.of(AEFeature.Core), EnumSet.noneOf(IntegrationType.class), PartQuartzFiber.class),
 
+    PartCreativeEnergy(690, EnumSet.of(AEFeature.Core), EnumSet.noneOf(IntegrationType.class),
+            PartCreativeEnergy.class),
```

Comment:
nit: Since this is no longer a modified quartz fiber, I think it makes more sense to put this line down at the bottom

Reply:
like this?